dart/utils/urdf/DartLoader.cpp

```diff
@@ -299,9 +300,12 @@ dynamics::Joint* DartLoader::createDartJoint(const urdf::Joint* _jt)
       joint = new dynamics::FreeJoint();
       break;
   case urdf::Joint::PLANAR:
-      std::cout << "Planar joint not supported." << std::endl;
-      assert(false);
-      return NULL;
+      joint = new dynamics::PlanarJoint();
+      // TODO(MXG): Should we read in position limits? The URDF limits
+      // specification only offers one dimension of limits, but a PlanarJoint is
+      // two-dimensional. Should we assume that position limits apply to both
```

Comment:
Actually, it's three-dimensional; x, y, and theta.

Let's don't accept the position limits for now. Can you create an issue for this?